M2/Macaulay2/packages/ReesAlgebra.m2

```diff
@@ -1256,10 +1278,8 @@ doc ///
 
 doc ///
   Key
-    normalCone
-    (normalCone, Ideal)
     (normalCone, Ideal, RingElement)
-    
+    (normalCone, Ideal)
```

Comment:
You can document `associatedGradedRing` and its methods here, since it is now missing documentation.

Reply:
By the way, did you intentionally flip the order of `(normalCone, Ideal)` and `(normalCone, Ideal, RingeElement)` methods in the list? This causes the headline to say:
> normalCone(Ideal,RingElement) -- The normal cone of a subscheme

which is unusual because the second parameter is not mentioned in the headline and is optional.